recipe/bld.bat

```diff
@@ -1,15 +1,16 @@
-python setup.py install
+%PYTHON% setup.py install
 if errorlevel 1 exit 1
 
 set MENU_DIR=%PREFIX%\Menu
 IF NOT EXIST (%MENU_DIR%) mkdir %MENU_DIR%
 
 copy %SRC_DIR%\img_src\spyder.ico %MENU_DIR%\
 if errorlevel 1 exit 1
-copy %SRC_DIR%\img_src\spyder_light.ico %MENU_DIR%\
-if errorlevel 1 exit 1
+REM copy %SRC_DIR%\img_src\spyder_light.ico %MENU_DIR%\
+REM if errorlevel 1 exit 1
```

Comment:
Also, please don't forget to remove these lines ;-)

They are not needed anymore.